src/app/modules/mpi/components/paciente-listado.html

```diff
@@ -41,6 +41,9 @@
         <tr class="hover" *ngFor="let paciente of listado"
```

Comment:
`<plex-list *ngIf="listado && listado.length">
    <plex-heading>
        <b label>Apellido y Nombre</b>
        <b label>Documento</b>
        <b label>Nacimiento</b>
    </plex-heading>
    <plex-item *ngFor="let paciente of listado" (click)="seleccionar(paciente)">
        <img class="img-thumbnail" [mpiFotoPaciente]="paciente" />
        <plex-label titulo="{{paciente | nombre}}" subtitulo=""></plex-label>
        <ng-container *ngIf="paciente?.documento !== ''">
            <plex-label titulo="{{paciente.documento}}"></plex-label>
        </ng-container>
        <ng-container *ngIf="paciente?.documento === ''">
            <plex-label titulo="N/D" type="danger"></plex-label>
        </ng-container>
        <ng-container *ngIf="paciente.fechaNacimiento !== null">
            <plex-label titulo="{{paciente.fechaNacimiento | fecha:'utc'}}" subtitulo="{{paciente | edad}}">
            </plex-label>
        </ng-container>
        <plex-badge type="info">{{paciente | sexo}}</plex-badge>
        <plex-badge *ngIf="paciente.estado == 'validado'" type="success">Validado</plex-badge>
        <plex-badge *ngIf="paciente.estado != 'validado'" type="warning">Temporal</plex-badge>
    </plex-item>
</plex-list>`